.github/actions/notify-PA-repo-of-new-version/action.yml

```
@@ -2,33 +2,43 @@ name: 'Notify PA repository of new BPMN visualization version'
 description: 'You can customize the parameters to link to the event and the repository to notify.
 To use on tag branch.'
 inputs:
-  PA_REPOSITORY:
-    description: 'The name of the repository on the process-analytics organization to notify'
+  TOKEN:
+    description: 'Token to dispatch on the repository'
     required: true
     default: ''
-  CUSTOM_PARAMETERS:
-    description: 'The custom parameters to link to the event to notify, with format: 
-    <, "parameter1": "value1">'
+  PA_REPOSITORY:
+    description: 'The name of the repository on the process-analytics organization to notify'
     required: true
     default: ''
-  TOKEN:
-    description: 'Token to dispatch on the repository'
-    required: true
+  BUILD_DEMO_WORKFLOW_ID:
+    description: 'The identifier of the workflow which is used to build the demo'
     default: ''
 
 runs:
   using: "composite"
   steps:
-    - id: get_version
-      name: Get the version
+    - name: Set env VERSION
```

Let's simply, we don't care in the logs that this is an environment variable or whatever else
```suggestion
    - name: Set VERSION
```